docs/autodoc_build/pypiper.md

```diff
@@ -318,7 +317,7 @@ it will first create a lock file to prevent other calls to run
 is being created. It also records the memory of the process and
 provides some logging output.
 ```python
-def run(self, cmd, target=None, lock_name=None, shell=None, nofail=False, clean=False, follow=None, container=None):
+def run(self, cmd, target=None, lock_name=None, shell=None, nofail=False, errmsg=None, clean=False, follow=None, container=None):
```

Comment:
No `errmsg` parameter now

Reply:
Ditto https://github.com/databio/pypiper/pull/135#discussion_r267857015

Reply:
ah yes, i keep forgetting that it documents the **installed** package and not the other... yeah it'll auto-update eventually :)